test/load/test_load_tx_ethereum.py

```diff
@@ -0,0 +1,273 @@
+import time
+import threading
+import copy 
+
+import siftool_path
+from siftool import eth, test_utils, sifchain
+from siftool.eth import NULL_ADDRESS
+from siftool.common import *
+from web3.eth import Contract
+import web3
+
+from eth_typing import (
+    ChecksumAddress,
+)
+
+# In separate window: test/integration/framework/siftool run-env
```

Comment:
Instead of having the user launch this is a separate window, I think it would make sense for the test suite to launch these things directly, and then shut them down when it is done.

Reply:
In most of case, we create the testing env and run lots of different test cases, integration test or load test. The startup needs quite a long time, so it is a waste of launching it to run a single test.